src/features/content-answers/ContentAnswersModal.scss

```diff
@@ -0,0 +1,49 @@
+@import '../../styles/variables';
+
+.ContentAnswersModal {
+    .modal-dialog-container {
+        display: flex;
+        flex-basis: 0;
+        width: 100%;
+        height: 100%;
+        max-height: 702px;
+    }
+
+    .modal-dialog {
+        width: 768px;
+        height: 100%;
+        max-height: 702px;
```

Comment:
were you able to check which `max-height` were needed and not?

my guess is you don't need `max-height` here or in `.modal-content`

Reply:
I checked that the height of modal-content will exceed 638px based on the screen/monitor size. so just in case, I put max-height for them